consensus/poet/core/sawtooth_poet/poet_consensus/poet_settings_view.py

```diff
@@ -16,12 +16,12 @@
 import math
```

Comment:
I prefer having commits that work, such that no commit causes a broken system.  Renaming these files is renaming the package.  I could see breaking the commit between renaming the class in the file, then the package name itself, but crafting that commit might be quite difficult at this point.